src/main/java/echo/Server.java

```diff
@@ -0,0 +1,91 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+
+
+public class Server {
+    public static void main(String[] args) throws IOException {
+        runEchoServer();
+    }
+
+    public static ServerSocket serverSocket = null;
+        public static Socket clientSocket = null;
+
+
+
+    public static void runEchoServer() throws IOException {
+
+        try {
+            serverSocket = createServerSocket(7777);
+
+        } catch (IOException e) {
+            errorMessage("Can't listen on port number 7777", e);
+
+        }
+
+        print("Listening for connection...");
+
+        try {
+
+            clientSocket = connectClient(serverSocket);
+
+        } catch (IOException e) {
+            errorMessage("***Accept failed!!!***", e);
+
```

Comment:
the indentation is very off in this file. Do you have a formatting tool for java?

Reply:
Yes I have  a formatting tool i will format it and push it up.